server/service/dashboard_server.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=MHP DAShboard server
+
+[Install]
+WantedBy=default.target
+
+[Service]
+Restart=on-failure
+# Note for this to work we need to run the below command from the DAShboard repository root directory, the Working directory is set by install.sh script
+ExecStart=/home/pi/.nvm/versions/node/v14.17.5/bin/node server.js
```

Comment:
This line could also be generated by the install script. In its current form it's very brittle relying on a specific installed node version.
Alternatively, if you want to just assume a path, perhaps `/usr/bin/node` is a more reasonable default. (`apt install node` will create this binary)

Reply:
Same issue as with previous comment in `dashboard_client.service`. 

The issue with using `/usr/bin/node` is that it may not use the correct node version, we need >14.0.0. Currently on the pi, node 14 is installed only for user pi. Systemd runs `ExecStart` as root user. My attempt to use `User=pi` in the systemd file didn't work. We can download latest version of node to root user, but that's going to be just as ugly of a dependency to have, unless it's automated.

Again, there may be a good solution, but I thought I shouldn't waste more time if I'm out of ideas. I'm still thinking of potential solutions, but thought I'll open this PR and maybe merge it for the sake of having a version of systemd files working.